apps/nextjs-app-router/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": ["next/babel","next/core-web-vitals"]
```

Comment:
Recommended ESLint config is now just `"extends": "next/core-web-vitals"`: https://nextjs.org/docs/app/building-your-application/configuring/eslint